operations/patch_util.go

```diff
@@ -30,8 +30,9 @@ const largePatchThreshold = 1024 * 1024 * 16
 // This is the template used to render a patch's summary in a human-readable output format.
 var patchDisplayTemplate = template.Must(template.New("patch").Parse(`
          ID : {{.Patch.Id.Hex}}
+    Project : {{.Patch.Project}}
```

Comment:
If I remember correctly, `(Patch).Project` stores the project ID (i.e. the `_id`), not its human-readable identifier. They can be used interchangeably for the most part, but I imagine we want to display the human-readable identifier here.

Reply:
<img width="1226" alt="image" src="https://github.com/evergreen-ci/evergreen/assets/26491602/b6121f97-e24b-4c74-b9d4-6e32f20faad3">
 
Updated to show identifier but I'm not sure if this might slow down the patch creation process because it makes an extra call to evergreen to get the project ref